### PR TITLE
Fixed #1552: Updated CONTRIBUTING.md

### LINKED ISSUE #1552: Update CONTRIBUTING.md
Add the missing important elements
I am working on it

### PR DESCRIPTION
**Pull request by samyakbvs**
Elements added to - 
* Pull Request Process
* Examples of behaviour that contributes to creating a positive environment
* Examples of unacceptable behavior by participants

### CODE REVIEW

**Comment by samyakbvs** [comment]
Closes #1552

**Comment by codedsun** [comment]
Was this needed?

**Comment by samyakbvs** [comment]
Yes @vaibhavsingh97

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,6 +34,7 @@ include:
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members
+* Always respect mentors time and don't nag them for reviews
```
Comment (harshit98) [inline CONTRIBUTING.md]: nag -> tag @samyakbvs

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ Please note we have a code of conduct, please follow it in all your interactions
 4. Wait for the maintainers to review your pull request and do the changes if requested.
 5. *We use danger, so update go to https://travis-ci.org and login with github. Next you will see a `+` near my repositories. Click on it and select your fork enabling it for Travis. Then go the dashboard and add two environment varibales which two wont get displayed in the terminal(There's an option). The first one must be named `USERNAME` and should contain your github username. The second one should be named `API_TOKEN` and should contain a github personal token with atleast scope to public repos. It enables fast and easy moderation adequate testing.*
 
+Note: Always squash your commits and provide a clear description of what does the particular Pull Request do
```
Comment (harshit98) [inline CONTRIBUTING.md]: Pull Request --> pull request

**Comment by harshit98** [comment]
@samyakbvs Do the suggested changes. 

@codedsun @vaibhavsingh97 Kindly review once the changes have been done! 💯

**Comment by samyakbvs** [comment]
Doing it @vaibhavsingh97

**Comment by anantprsd5** [comment]
@samyakbvs Most of the required changes have already been done in the PR #1549. Closing this PR.  Thanks.

**Pull request closed without merging**